python-lib/dku_deeplearning_image/utils.py

```diff
@@ -10,6 +10,7 @@
 from collections import OrderedDict
 import numpy as np
 from datetime import datetime
+import GPUtil
```

Comment:
Code structure suggestion, why not put all the GPU-related functions from utils.py in GPUtil?

Reply:
Actually, GPUUtil is an external library

Reply:
Ah ok. Maybe just rename your module to gpu_utils.py? Assuming it contains only gpu related stuff

Reply:
It does not, it has all utils functions for my Plugin. Do you think I should separate it?

Reply:
Not a big deal, but yeah, I think your GPU related functions are interesting enough to have their own module